libs/trainer/tests/test_wrapper.py

```diff
@@ -43,10 +42,10 @@ def to(self, device):
         return
 
 
-class Preprocessor(Transform):
+class Preprocessor(torch.nn.Module):
```

Comment:
Wondering why this was a `Transform`

Reply:
Don't think it needed to be. If you're curious, the concept has been moved to `ml4gw` and is now called `FittableTransform`

Reply:
Fittable transform seem like a better option here, with a `build` function etc. no?